src/pages/messages/components/NviCandidatePage.tsx

```diff
@@ -54,22 +49,45 @@ export const NviCandidatePage = ({ nviListQuery }: NviCandidatePageProps) => {
     meta: { errorMessage: t('feedback.error.get_registration') },
   });
 
-  const periodStatus = nviCandidate?.periodStatus.status;
+  const nviListQuery = location.state?.nviQuery;
```

Comment:
om tanken er å erstatte alle `nviListQuery` med `nviQuery` anbefaler jeg å gjøre et globalt søk. ellers er det vanskelig å fange det opp i alle filer det gjelder
```suggestion
  const nviQuery = location.state?.nviQuery;
```

Reply:
Ble litt surr her mellom state og verdi, Gikk får å bruke `nviQuery` i state, og `nviListQuery` som props, da det er det verdien heter i utgangspunket.